modules/ding_mobilesearch/ding_mobilesearch.module

```diff
@@ -353,8 +353,21 @@ function ding_mobilesearch_ding_mobilesearch_node_export_mapping($node) {
       ->fields('oh')
       ->condition('oh.nid', $node->nid)
       ->execute();
+
+    // Push only opening hours from current week till 10 weeks ahead.
+    $current_date = date('Y-m-d', REQUEST_TIME);
+    // Align with start of the week.
+    $current_day = date('N', REQUEST_TIME) - 1;
+    $now = strtotime($current_date) - $current_day * 86400;
+    // 10 weeks.
+    $threshold = 6048000;
+
```

Comment:
I think we can let PHP do the heavy lifting and reduce this to 

```php
strtotime('last monday + 10 weeks');
```
https://3v4l.org/Z2r2p

I think that makes it a lot clearer what the threshold is.

An explanation of why we need this threshold would also be good.